nodejs/eks/examples/examples_test.go

```diff
@@ -47,6 +49,30 @@ func Test_Examples(t *testing.T) {
 				"@pulumi/eks",
 			},
 			ExpectRefreshChanges: true,
+			ExtraRuntimeValidation: func(t *testing.T, info integration.RuntimeValidationStackInfo) {
```

Comment:
This looks like the same copy-pasted function several times. Can you define the function once and just reference it here?

Reply:
Done